.github/workflows/nightly-docs.yml

```diff
@@ -0,0 +1,39 @@
+name: Nightly documentation
+
+on: [push]
+
+jobs:
+    docs:
+        if: "! contains(toJSON(github.event.commits.*.message), '[ci skip]')"
+        runs-on: ubuntu-latest
+        steps:
+            - name: Configure workflow
+              id: configuration
+              run: |
+                echo "DOCS_OUTPUT_DIR=${GITHUB_WORKSPACE}/skript-docs/docs/nightly/${GITHUB_REF#refs/heads/}" >> $GITHUB_OUTPUT
+                echo "DOCS_REPO_DIR=${GITHUB_WORKSPACE}/skript-docs" >> $GITHUB_OUTPUT
+                echo "SKRIPT_REPO_DIR=${GITHUB_WORKSPACE}/skript" >> $GITHUB_OUTPUT
+            - name: Checkout Skript
+              uses: actions/checkout@v3
+              with:
+                  submodules: recursive
+                  path: skript
+            - name: Setup documentation environment
+              uses: ./skript/.github/workflows/docs/setup-docs
+              with:
+                  docs_deploy_key: ${{ secrets.DOCS_DEPLOY_KEY }}
+                  docs_output_dir: ${{ steps.configuration.outputs.DOCS_OUTPUT_DIR }}
+            - name: Generate documentation
+              uses: ./skript/.github/workflows/docs/generate-docs
+              with:
+                  docs_output_dir: ${{ steps.configuration.outputs.DOCS_OUTPUT_DIR }}
+                  docs_repo_dir: ${{ steps.configuration.outputs.DOCS_REPO_DIR }}
+                  skript_repo_dir: ${{ steps.configuration.outputs.SKRIPT_REPO_DIR }}
+            - name: Push nightly documentation
+              uses: ./skript/.github/workflows/docs/push-docs
+              with:
+                  docs_output_dir: ${{ steps.configuration.outputs.DOCS_OUTPUT_DIR }}
+                  docs_repo_dir: ${{ steps.configuration.outputs.DOCS_REPO_DIR }}
+                  git_name: Nightly Docs Bot
+                  git_email: nightlydocs@skriptlang.org
+                  git_commit_message: Update nightly docs
```

Comment:
Could this grab the commit it's generating for? to be like `Update nightly docs on COMMIT_LINK`

Reply:
sure. do you want the commit link or the short commit sha?

Reply:
The link would be good, github will rename it to commit id and it will be clickable for reference 👏